pabc.client/src/router/index.ts

```diff
@@ -1,6 +1,7 @@
 import { createRouter, createWebHistory } from "vue-router";
 import DashboardView from "@/views/DashboardView.vue";
 import AdminView from "@/views/AdminView.vue";
+import UnauthorizedView from "@/views/UnauthorizedView.vue";
 
 const router = createRouter({
```

Comment:
kijk even goed naar de routes van ita. je moet per route aangeven of je ervoor ingelogd moet zijn en of je er de rol voor moet hebben